scheduler-interface-spec.md

```diff
@@ -505,8 +505,6 @@ message Allocation {
   Resource resourcePerAlloc = 5;
   // Priority of ask
   int32 priority = 6;
-  // Queue which the allocation belongs to
-  string queueName = 7;
```

Comment:
Can we reserve this ID and name so it cannot be reused?
```
reserved 7;
reserved "queueName"
```
This will prevent re-using the same ID when we update later, which can cause all kinds of issues. The reserved lines can be  used to replace the existing lines in the message or at the start/end of the message.

NIT: now that we are making a change here fixing the indentation of the lines 505 to 510 might be nice too.

Reply:
Thanks for your review.